src/test/resources/features/account/search_account.feature

```diff
@@ -0,0 +1,12 @@
+@Account
+Feature: Search Account
+
+  @deleteAccount @createAccount
+  Scenario: Account search using the search tool and compare with the account created
+    Given I log in to Salesforce with Account Owner User credentials
+    When I go to "HOME" page
+    And I search the following name
+      | Armando Guerra |
+    Then I search these user name in the drop down menu
+    And I select the user name
+    And I validate the information
```

Comment:
You need to be more specific, which information you have to validate? Maybe this step can be "the Account's information should be displayed at details" (avoiding to use validate).